### DISCUSSION TITLE
Dernek ilanlarinin tek menu altında toplanmasi

### DISCUSSION THREAD

**Discussion by snmclkyp** [acikkaynak/deprem-yardim-projesi#111] [Ideas]
Birçok dernek katkida bulunmak amaci ile duyuru yayimliyor
Orn; kanser derneği, kanser hastalarina destek olacağını duyurmuş. Bu duyurular su an dağınık, bunları bir menu altında toplasak?

**Comment by sercangoger** [comment]
STK,dernekler bu tart bütün yardım kuruluşlarının faaliyetleri konularının anlatıldığı bilindiği ve bu tarz durumlar da bildirimlerini sisteme girip gidecekleri yeri söylemeleri koordineyi yönetmek adına örnek oluşturacaktır.
